pkg/controller/revision/revision_controller.go

```diff
@@ -256,12 +260,30 @@ func (r *ReconcileRevision) Reconcile(request reconcile.Request) (reconcile.Resu
 
 	instance.Status = status
 	if err = r.client.Status().Update(context.TODO(), instance); err != nil {
-		return reconcile.Result{}, err
+		return r.Requeue(log, err)
 	}
 
+	return r.Requeue(log, nil)
+}
+
+func (r *ReconcileRevision) Requeue(log logr.Logger, err error) (reconcile.Result, error) {
+	if err != nil {
+		log.Error(err, "Reconcile resulted in error")
+		return reconcile.Result{}, err
+	}
+	log.Info("Reconciled successfully", "requeueAfter", r.config.RequeueAfter)
 	return reconcile.Result{RequeueAfter: r.config.RequeueAfter}, nil
 }
 
+func (r *ReconcileRevision) NoRequeue(log logr.Logger, err error) (reconcile.Result, error) {
+	if err != nil {
+		log.Error(err, "Reconcile resulted in error")
+		return reconcile.Result{}, err
+	}
+	log.Info("Reconciled successfully, no requeue")
```

Comment:
```suggestion
	log.Info("Reconciled successfully", "requeue", false)
```

will make these logs more searchable